mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -171,6 +171,25 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 		ARG 0 values
 	METHOD method_29715 stream (Lnet/minecraft/class_238;)Ljava/util/stream/Stream;
 		ARG 0 box
+	METHOD method_30512 iterateInSquare (Lnet/minecraft/class_2338;ILnet/minecraft/class_2350;Lnet/minecraft/class_2350;)Ljava/lang/Iterable;
```

Comment:
Is this horizontal or vertical? From your javadoc and name I cannot determine if it is a totally horizontal iteration.

Reply:
Depends on the directions passed to this method. If both directions are horizontal than the square is horizontal too.

Reply:
Hmm, so this can be either a vertical or a horizontal square, and the third axis is just ignored? Guess square is fine as square is 2d.